script.js

```diff
@@ -1,8 +1,62 @@
-const elements = document.querySelectorAll(".item");
+const container = document.querySelector(".container__content");
 
-elements.forEach(element => {
-    const checkbox = element.querySelector("input");
-    checkbox.addEventListener('click', () => {
-        element.classList.toggle("checked-accept");
+function render(e) {
+    e.forEach(toDo => {
```

Comment:
```suggestion
    todos.forEach(todo => {
```

Reply:
The variable "toDo" stands for To Do List.